packages/block-library/src/cover/deprecated.js

```diff
@@ -244,6 +244,157 @@ const v12BlockSupports = {
 	},
 };
 
+// Deprecation for blocks that have z-index.
+const v14 = {
```

Comment:
Thank you for the suggestion and the example! ♥️
I'll try to fix it today!

Reply:
I followed [this documentation](https://github.com/WordPress/gutenberg/blob/trunk/test/integration/fixtures/blocks/README.md), and based on your example, I added the new fixture: https://github.com/WordPress/gutenberg/pull/66249/commits/9ba72c00bdebe83eaf5e844e2b9fe178096d3849 (created the HTML and let the command generate the others)

I'm still struggling a bit to fix the others. I let you know here when it's done.

Reply:
I could make it work now with the fixture fixes (https://github.com/WordPress/gutenberg/pull/66249/commits/0ae62731398295ede211eb9d199b4853ddcfea0c), but when running the `fixtures:regenerate` again, it's trying to add the `'has-no-zindex-in-inner-blocks'` as a `className` attribute.

Not sure if it's a bug with the `fixtures:regenerate` command, or if adding a class to the wrapper element wouldn't be recommended. 🤔

I thought of not using the sibling class, as you suggested to fix this, but it would mean a few new classes instead of just one. Another approach would be adding an additional wrapper with a class, it would break a few styles (we have some using `>` that we would need to refactor to do it, so I think it would be risky).

I'm out of ideas now. I need to think more.

Reply:
Hey! 👋

Thank you for your suggestions! I followed your suggestion and applied it with this commit: https://github.com/WordPress/gutenberg/pull/66249/commits/0e25fed09bdf79a48f0f32a678c36da5a19d056e.